readthedocs/doc_builder/environments.py

```diff
@@ -102,7 +102,7 @@ def __init__(
     ):
         self.command = command
         self.shell = shell
-        self.cwd = cwd or os.getcwd()
+        self.cwd = cwd or '$HOME'
```

Comment:
I wonder if we're depending on the old default somewhere... hopefully not.

Reply:
If we were, it would have been an error on our builders at least p:, we are expanding that var for people that aren't using docker.

Reply:
greped for `.run(`, we are setting the cwd explicitly on all of them.